docker-compose.yml

```diff
@@ -8,7 +8,4 @@ services:
         ports:
             - 8888:8888
         volumes:
```

Comment:
Wahrscheinlich gibt es durch WSL ein Problem, bei dem bestimmte Datei Permissions nicht richtig gesetzt werden. Dort entsteht dann folgender Fehler:
`RuntimeError: Permissions assignment failed for secure file: '/home/jovyan/.local/share/jupyter/runtime/kernel-9e25187f-fa92-4283-88fc-242c3ed1651e.json'. Got '0o677' instead of '0o0600'.`

Dieser Fehler kann durch setzten der Umgebungsvariable `JUPYTER_ALLOW_INSECURE_WRITES=true` umgangen werden, siehe [hier](https://stackoverflow.com/a/60821998).

Reply:
```yaml
version: '3.8'

services:
    jupyter_container:
        build:
            context: .
        init: true
        ports:
            - 8888:8888
        environment:
            - JUPYTER_ALLOW_INSECURE_WRITES=true
        volumes:
            - ./:/home/jovyan/
```

Reply:
habs angepasst